src/Components/Pollutantgraph.jsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Instead of disabling the linter checks, I would strongly recommend you fix the linter error.